thewalrus/tests/test_internal_modes.py

```diff
@@ -1419,7 +1432,10 @@ def test_density_matrix(cutoff):
     rho2 = density_matrix_single_mode(cov, N, cutoff=cutoff)
     rho2_norm = rho2 / np.trace(rho2).real
 
+    # probs = probabilities_single_mode(cov, N, cutoff=cutoff, normalize=True)
```

Comment:
@DavidSPhillips this is a waaay too long test..

Reply:
well, it takes ages on my computer, that is why I decided against adding the new functions in this test

Reply:
What's too long? The whole test or the addition of the commented out probabilities bit? The new bit doesn't take much longer for my laptop at all (after correcting for the PNR on mode 0). The whole test takes about 2.5 minutes.

Reply:
The test. 2.5 minutes is too long.

Reply:
But the test is still included so it will still take 2.5 minutes. Adding the commented out lines (but not commented out) wouldn't increase the time much.